HA for VNets w/ ER

jaredr80 opened the pull request:
Adding additional FAQ to reflect HA capabilities with ExpressRoute on a single GW

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jaredr80 committed:
HA for VNets w/ ER

Adding additional FAQ to reflect HA capabilities with ExpressRoute on a single GW

jaredr80 committed:
Update expressroute-faqs.md

jaredr80 reviewed (commented):
Updated with Charley's comments
